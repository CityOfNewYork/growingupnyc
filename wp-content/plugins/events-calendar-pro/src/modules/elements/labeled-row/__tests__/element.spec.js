import React from 'react';
import renderer from 'react-test-renderer';
import LabeledRow from '../element';

describe( 'LabeledRow', () => {
	test( 'should render component', () => {
		const component = renderer.create(
			<LabeledRow
				className="test-class"
				label="hello"
			>
				<div>Test</div>
			</LabeledRow>
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
