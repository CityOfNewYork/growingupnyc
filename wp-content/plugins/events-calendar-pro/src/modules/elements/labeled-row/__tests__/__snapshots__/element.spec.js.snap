// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LabeledRow should render component 1`] = `
<div
  className="tribe-editor__events-pro__row tribe-editor__labeled-row test-class"
>
  <div
    className="tribe-editor__events-pro__label tribe-editor__labeled-row__label"
  >
    <span
      className="tribe-editor__events-pro__label-text"
    >
      hello
    </span>
  </div>
  <div
    className="tribe-editor__labeled-row__content"
  >
    <div>
      Test
    </div>
  </div>
</div>
`;
